hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -1597,6 +1597,11 @@ KeepAliveCache getKeepAliveCache() {
     return keepAliveCache;
   }
 
+  @VisibleForTesting
+  Timer getTimer() {
```

Comment:
Let's have this package protected.
Unit test accessing this will still be able to access.